Doc/whatsnew/3.14.rst

```diff
@@ -1072,6 +1072,14 @@ io
   :gh:`120754` and :gh:`90102`.)
 
 
+textwrap
+--------
+
+* Optimise the :func:`~textwrap.dedent` function, improving performance by
+  an average of 2.4x, with larger improvements for bigger inputs,
+  and fix a bug with incomplete normalization of blank lines with whitespace
+  characters other than space and tab.
+
```

Comment:
Add something like (to be able to see the issue)

```rst
  characters other than space and tab.
  (Contributed by [...] in :gh:`...`.)

```

+ 2 blank lines to end the section.